extensions/eclipse-che-theia-plugin-remote/src/node/hosted-plugin-process-log.ts

```diff
@@ -33,14 +33,16 @@ export class LogHostedPluginProcess extends HostedPluginProcess {
 
     if (childProcess) {
       if (childProcess.stdout) {
-        childProcess.stdout.on('data', data =>
-          client.log({ data: `Extension-Host:${data.toString().trim()}`, type: LogType.Info })
-        );
+        childProcess.stdout.on('data', data => {
```

Comment:
how these changes are related ? we don't have then twice the messages ?

Reply:
One of the reasons no-one noticed the exception at startup is that the output of the child process is not visible when doing self-hosting, except if the the developer happens to view the output view. 
So this change would make it more likely that a developer introducing a change would detect that it breaks something.
As for duplicates message: I haven't noticed that the output view has duplicates.